src/lib/features/playback-controls/model/toggle-global-playback.ts

```diff
@@ -3,6 +3,10 @@ import { play, pause, audio } from '$lib/entities/audio';
 
 export function toggleGlobalPlayback() {
   const $audio = get(audio);
+  if ($audio === undefined) {
+    return;
+  }
```

Comment:
Question: why are these three lines of code everywhere? Is there no way to ensure the `$audio` is not `undefined`?

Reply:
Well, that's the only way)